app/feature/feature-travel-certificate/src/main/kotlin/com/hedvig/android/feature/travelcertificate/navigation/TravelCertificateGraph.kt

```diff
@@ -27,6 +27,7 @@ fun NavGraphBuilder.travelCertificateGraph(
   navigator: Navigator,
   applicationId: String,
   onNavigateToCoInsuredAddInfo: (String) -> Unit,
+  onNavigateToAddonPurchaseFlow: (List<String>) -> Unit,
```

Comment:
If we need this to only be represented by a non-empty list of IDs, as you note in some comment else in the code, could this be a NonEmptyList instead?
If yes, can we still pass that as expected through Koin etc?
Just an idea, not necessary to do it if it complicates things too much.

Reply:
will try that!

Reply:
I think what may be better to do instead is to make TravelAddonBannerInfo eligibleInsurancesIds property a NonEmptyList, since this will be the source of banner everywhere. Then we won't show the banner at all and log the empty list as error if it would come from be

Reply:
Yeah that sounds good, do give it a try if you find the time :)